assets/css/dashboard.scss

```diff
@@ -64,13 +64,20 @@ html, body {
     flex: 1;
     -webkit-flex-direction: column;
     flex-direction: column;
-    > h3, > h4 {
+    > h4 {
```

Comment:
We should avoid selectors that target HTML elements and bootstrap classes.

You could add on the folder list-items : 
```css
cursor: grab;
-webkit-cursor: grab;
```